fintrack/bank_accounts/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+from banks.models import BankBranch
+from users.models import User
+
+
+class BankAccount(models.Model):
+    class AccountTypeChoices(models.TextChoices):
```

Comment:
create a choices.py file and move it there
it will allow you to easily access choices in other files